### ISSUE TITLE
Add save_to_fits method to Marvin-tools classes

### ISSUE THREAD

**Issue by bretthandrews** [marvin-manga/marvin_public_old#4]
Motivated by a question from @margudo, it would be helpful to have a method to save user-modified data to a similar format as the input data (i.e., FITS files containing cubes or maps or spectra).

**Comment by albireox** [comment]
This is a good idea but not one trivial to implement, so I would push it to a later tier. We'd also need to define what data we want to save. For instance, I see the point in wanting to save Map objects (or the ratio of two Map), but other classes may be not so useful and more complicated to implement.

**Comment by bretthandrews** [comment]
Fair enough. Perhaps an easy temporary solution is to include some save snippets in the examples (e.g., for plots).

**Comment by bretthandrews** [comment]
OK. There's now a snippet to save the spectrum that is plotted in the First Steps page in the Sphinx docs, and we should add something similar to the corresponding Maps, Spectrum, etc. pages when we make them.

**Comment by albireox** [comment]
We discussed this and decided to implement save Maps and ratio maps to FITS in a (at least initially) simple way: a FITS file with one data extension and limited metadata. For plots, we'll implement the save to PDF and PNG (basically the `plt.savefig` method).
